HeterogeneousCore/AlpakaTest/plugins/alpaka/TestAlgo.dev.cc

```diff
@@ -30,7 +30,27 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
       }
 
       // make a strided loop over the kernel grid, covering up to "size" elements
-      for (int32_t i : elements_with_stride(acc, size)) {
+      for (int32_t i : elements_with_stride(acc, view.metadata().size())) {
+        view[i] = {0., 0., 0., i, matrix * i};
+      }
+    }
+  };
+
+  class TestAlgoMultiKernel {
+  public:
+    template <typename TAcc, typename = std::enable_if_t<is_accelerator_v<TAcc>>>
+    ALPAKA_FN_ACC void operator()(TAcc const& acc, portabletest::TestDeviceMultiCollection::View<1> view) const {
```

Comment:
Since the `view()` can be called with a type argument, I think it would be neat to be able to use the `View` alias with type arguments as well. I'm not sure of its feasibility though (I can imagine supporting both index and type arguments there could be complicated), but I want to throw out the idea anyway.